android/res/layout/routing_plan_details.xml

```diff
@@ -7,8 +7,10 @@
   style="@style/MwmWidget.Floating.Panel"
   android:layout_width="match_parent"
   android:layout_height="@dimen/routing_selector_size"
-  android:paddingLeft="8dp"
-  android:paddingRight="12dp"
+  android:paddingLeft="@dimen/margin_half"
+  android:paddingStart="@dimen/margin_half"
+  android:paddingRight="@dimen/margin_half_plus"
+  android:paddingEnd="@dimen/margin_half_plus"
```

Comment:
+, спасибо.
> On May 25, 2016, at 7:56 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/routing_plan_details.xml <https://github.com/mapsme/omim/pull/3340#discussion_r64608841>:
> 
> > @@ -7,8 +7,10 @@
> >    style="@style/MwmWidget.Floating.Panel"
> >    android:layout_width="match_parent"
> >    android:layout_height="@dimen/routing_selector_size"
> > -  android:paddingLeft="8dp"
> > -  android:paddingRight="12dp"
> > +  android:paddingLeft="@dimen/margin_half"
> > +  android:paddingStart="@dimen/margin_half"
> > +  android:paddingRight="@dimen/margin_half_plus"
> > +  android:paddingEnd="@dimen/margin_half_plus"
> Ты забыл про layout-w1020dp/routing_plan_details.xml
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3340/files/0eaf8797da6003592602a2144d0bd41203a7c93b..fb35b52ab26a99751d3582d9e6e9c073e8d3b3fe#r64608841>